test/unit/pipeline/test_optimize_hyperparameters.py

```diff
@@ -65,6 +68,25 @@ def test_optimize_hyperparameters_pipeline_no_load_data(self):
                 self.pj, input_data, "./test/unit/trained_models", n_trials=2
             )
 
+    @patch("openstef.model.serializer.MLflowSerializer.save_model")
+    def test_optimize_hyperparameters_pipeline_quantile_regressor(
+        self, save_model_mock
+    ):
+        """If the regressor can predict quantiles explicitely,
+        the model should be retrained for the desired quantiles"""
+        pj = self.pj
+        predefined_quantiles = (0.001, 0.5)
+        pj["quantiles"] = predefined_quantiles
+        pj["model"] = "xgb_quantile"
+
+        parameters = optimize_hyperparameters_pipeline(
+            pj, self.input_data, "./test/unit/trained_models", n_trials=2
```

Comment:
Doesnt this slow down the unit tests by a lot? Even with 2 trials?

Reply:
Good idea, I reduced it to 1